src/main/java/hudson/plugins/git/UserRemoteConfig.java

```diff
@@ -131,7 +131,7 @@ public FormValidation doCheckUrl(@AncestorInPath Item project,
                                          @QueryParameter String credentialsId,
                                          @QueryParameter String value) throws IOException, InterruptedException {
 
-            if (project == null || !project.hasPermission(Item.CONFIGURE)) {
+            if (project == null || !project.hasPermission(Item.EXTENDED_READ)) {
```

Comment:
(may be occasions where `USE_OWN` is valid, but pending JENKINS-31870 this does not work anyway: is checking `SYSTEM`)